### PR TITLE
Add supports for memory related tests

### PR DESCRIPTION
**Pull request by Yingshun**
1. Add a function to compare memlock
2. Add a function to normalize mem size

Signed-off-by: Yingshun Cui <yicui@yicui.nay.csb>

Test results:
https://github.com/autotest/tp-libvirt/pull/4702#issue-1526662137

### CODE REVIEW

**Comment by Yingshun** [comment]
@nanli1  Could you plz review this PR? Thanks!

**Review by cliping — APPROVED** [review]
LGTM

**Review by nanli1 — APPROVED** [review]

File: virttest/utils_libvirt/libvirt_memory.py [hunk 1]
```diff
@@ -6,15 +6,45 @@
...
 
+def comp_memlock(exp_memlock):
+    """
+    Compare the locked mem with the given value.
+
+    :param exp_memlock: The expected locked mem value
+    :raise: TestError if the actual locked mem is invalid
+    :return: True on success
+    """
+    logging.debug("Check if the memlock is %s.", exp_memlock)
```
Comment (chloerh) [inline virttest/utils_libvirt/libvirt_memory.py:23]: LOG.debug()

File: virttest/utils_libvirt/libvirt_memory.py [hunk 2]
```diff
@@ -6,15 +6,45 @@
...
+    tmp_act_memlock = get_qemu_process_memlock_hard_limit()
+    logging.debug("Actual memlock is {}.".format(tmp_act_memlock))
```
Comment (chloerh) [inline virttest/utils_libvirt/libvirt_memory.py:25]: LOG.debug()

File: virttest/utils_libvirt/libvirt_memory.py [hunk 3]
```diff
@@ -6,15 +6,45 @@
...
+
+def normalize_mem_size(mem_size, mem_unit):
+    """
+    Normalize the mem size and convert it to bytes.
+
+    :param mem_size: The mem size
+    :param mem_unit: The mem size unit
+    :return: Byte format size
+    """
+    return mem_size*1024**['B', 'K', 'M', 'G', 'T'].index(mem_unit[0].upper())
```
Comment (chloerh) [inline virttest/utils_libvirt/libvirt_memory.py:50]: This function is really helpful! 
Do we consider a type check on `mem_size` or `int(mem_size)`

File: virttest/utils_libvirt/libvirt_memory.py (hunk 1 above)
Comment (Yingshun) [reply virttest/utils_libvirt/libvirt_memory.py]: THX, updated.

File: virttest/utils_libvirt/libvirt_memory.py (hunk 2 above)
Comment (Yingshun) [reply virttest/utils_libvirt/libvirt_memory.py]: updated.

File: virttest/utils_libvirt/libvirt_memory.py (hunk 3 above)
Comment (Yingshun) [reply virttest/utils_libvirt/libvirt_memory.py]: Yes, you're right, code updated.

**Review by chloerh — APPROVED** [review]

**Pull request merged**